less/home-page.less

```diff
@@ -56,20 +68,106 @@
       p {
         margin-bottom: 26px;
       }
-
-      .btn {
-      }
+      
+      
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
+      max-width: 100%;
       
       img {
-        width: 100%;
+       width: 100%;
         height: auto;
         border-radius: 10px;
       }
     }
+    
+  }
+   // added mixins to my button
+  .btn{
+    .button();
+    animation-name: spin;
+    animation-duration: 8s;
+
+  }
+  // added animation
+  @keyframes spin{
+    from{transform: rotate(0deg);}
+    to{transform: rotate(360deg);}
+}
+  
+}// home
+
+
+// media queries start here
+@media @mobile{
+  
+  .walk-img {
+     
+   width: 100%;
+    
+    
+   #walk-img{
```

Comment:
Why are you doubling up here on the `walk-img` using both a class selector and id selector on what seems to be the same element, and passing it the same `width` property in both cases?